cloud9_config.py

```diff
@@ -0,0 +1,123 @@
+""" Script to simplify the Cloud9 IDE configuration. 
+This works together with the CloudFormation template used to seed the workshop and a `roboMakerSettings.json` configuration file supplied
+as part of this project. Any changes to those will require reviewing this script
+"""
+
+import os
+import json
+import argparse
+import re
+import shutil
+
+import boto3
+
+LAUNCH_FILE_PATH = 'robot_ws/src/martian_detector/launch/martian_detector.launch'
+CONFIG_FILE_NAME = 'roboMakerSettings.json'
+SIM_CONFIG_ID = 'sim-robot-with-simple-world'
+BUCKET_KEY = 'RoboMakerS3Bucket'
+CONFIG_KEYS = {
```

Comment:
In us-east-1, RoboMaker is only available in 3 AZs. For this reason, we left the VPC section blank in this introductory lab to ensure there are no launching issues in this region. us-east-1 is the default region for AWS Educate, meaning folks may run into these issues. Can you add flag to "skip VPC" and then remove the VPC block in the roboMakerSettings.json file?

Reply:
The main reason for adding VPC block was that in my experiments access to S3 or Rekognition did not work without it. Is it possible to have Public IP without VPC attachment?